site/styles/_forms.scss

```diff
@@ -88,20 +91,41 @@ s72-activatedevice-form,
   .s72-datepicker-year {
     flex-shrink: 0;
   }
+  .s72-datepicker-month {
+    margin: 0 4px;
+    width: 100%;
+  }
 }
 // Form Control Default
 .s72-form-control {
   font-weight: $font-weight-normal;
+  height: var(--input-height);
+
+  /* stylelint-disable-next-line */
+  &:not(.StripeElement--empty) {
```

Comment:
I know this PR is closed but this causes the stripe input field to turn black with black text when you start typing

Reply:
Oof... I'll take care of that.

Reply:
Fix in develop now!